public/bars.js

```diff
@@ -304,7 +304,9 @@ function applyWaveAnimation() {
                 children[child].classList.remove("wave-visual");
     
                 children[child].style.display = "inline-block";
-                children[child].style.animation = `${waveSpeed}s wave cubic-bezier(0.37, 0, 0.63, 1) infinite ${-child * wavelength}s alternate`;
+                // if live editing a page to change all cubic-bezier to linear this looks weird but hopefully it works fine
+                // linear is more accurate to the game (with an amplitude of 15-20ish % ?
+                children[child].style.animation = `${waveSpeed}s wave linear(0.37, 0, 0.63, 1) infinite ${-child * wavelength}s alternate`;
```

Comment:
![firefox_4nwWLUk7Bo](https://github.com/user-attachments/assets/670176c8-cbac-4c99-8b67-e1b4527b8519)

not sure exactly what you were going for so i can't fix it but i don't think this was it lol

Reply:
oops the numbers in the brackets do mean something actually. should be fixed in commit [1e1649a](https://github.com/felikatze/isat-script-project/pull/226/commits/1e1649a3323c63a3cc0ee7f60d8c4ec3bb7f8e42)?

Reply:
yep, looks good! in the cubic-bezier version the numbers are the xy coordinates of the two handles, but in the linear version they seemed to be where the points were at a given point